npm_and_yarn/helpers/package.json

```diff
@@ -13,7 +13,6 @@
     "@npmcli/arborist": "^2.2.2",
     "detect-indent": "^6.0.0",
     "npm6": "npm:npm@6.14.11",
-    "npm7": "npm:npm@7.4.0",
```

Comment:
💯  nice not having two different versions of npm 7 installed 🙌

We could also change `npm6` back to `npm` which would allow dependabot to keep it up to date as I don't think it currently handles aliased packages

Reply:
Yeah, good call 👍

Reply:
Not blocking to get this out, seems like a good follow up to change `npm6` across the helpers